apps/dashboard/app/javascript/files/data_table.js

```diff
@@ -222,6 +222,9 @@ class DataTable {
                     data: null,
                     orderable: false,
                     defaultContent: '<input type="checkbox">',
+                    render: (data, type, row, meta) => {
+                        return "<input type='checkbox' data-dl-url='" + row.download_url + "'>";
```

Comment:
Can we use string interpolation here instead? I tend to get turned around by all the quotations in `+` expressions.

```javascript
return `<input type='checkbox' data-dl-url='${row.download_url}'>`;
```